examples/proto/examplepb/BUILD.bazel

```diff
@@ -38,10 +38,7 @@ proto_library(
 
 go_proto_library(
     name = "examplepb_go_proto",
-    compilers = [
-        "@io_bazel_rules_go//proto:go_grpc",
-        "//protoc-gen-grpc-gateway:go_gen_grpc_gateway",
```

Comment:
I think this line being removed is what is causing you issues. The way it is configured right now, you've only got the grpc protoc plugin, you also need the grpc-gateway protoc plugin.

If this was removed by Gazelle, then you can tag it with a `#  keep` after the line and it'll be happy. The full example would be

```
go_proto_library(
    name = "examplepb_go_proto",
    compilers = [
        "@io_bazel_rules_go//proto:go_grpc",
        "//protoc-gen-grpc-gateway:go_gen_grpc_gateway", #  keep
    ],
    importpath = "github.com/grpc-ecosystem/grpc-gateway/examples/proto/examplepb",
    proto = ":examplepb_proto",
...
```

Reply:
Great, I'll try that